src/scoreCard.js

```diff
@@ -0,0 +1,83 @@
+class ScoreCard {
+  constructor(frames, frame, score) {
+    this.frames = []
+    this.frame = null
+    this.lastFrame = null
```

Comment:
Assigning null instead of leaving a var undefined is good, it means you can test for it later too with ===. 

worth thinking about because var == null and var == undefined are the same whereas var === null and var === undefined aren't!